h/static/scripts/group-forms/utils/api.ts

```diff
@@ -65,6 +65,20 @@ export type PaginatedResponse<Item> = {
  */
```

Comment:
We now have `utils/api.ts` and a `utils/api` directory. It may be unclear whether an import for `utils/api` is going to import `utils/api.ts` or `utils/api/index.ts` (which doesn't exist). There will be a rule that prefers one of these, but if `api.ts` was moved to `api/index.ts` that would avoid the confusion.

Reply:
Good point.

> if `api.ts` was moved to `api/index.ts` that would avoid the confusion.

I'll do that.